guide/topics/webdev/reference.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Reference
+---
+You can find a reference to what each piece of HTML and CSS can do.
+
+# HTML
+
+## HTML Doctype Declaration
+
+#### Reason - 
+To Declare HTML Documents.
+
+#### Usage - 
+`<!DOCTYPE html>`
+
+[Guide Position](fullguide.html#starting-code)
+
+## Html
+
+#### Reason - 
+The Complete Encasing Tag for HTML Documents, All of your HTML Code goes in this tag.
+
+#### Usage - 
+`<html>(tags)</html>`
+
+## Head
+
+#### Reason - 
+To Store Metadata for the Website such as the title or importing CSS Sheets.
+
+#### Usage - 
+`<head>(tags)</head>`
+
+## Title
+
+#### Reason - 
+Defines the text that appears on the tab of the page.
+
+#### Usage - 
+`<title>(text)</title>`
+
+## Body
+
+#### Reason - 
+Where all your code that is displayed to the user should go.
+
+#### Usage - 
+`<body>(tags)</body>`
+
+## Paragraph
+
+#### Reason - 
+Shows text in a paragraph format
+
+#### Usage - 
+`<p>(text)</p>`
+
+## Headings
+
+#### Reason - 
+Puts emphasis on different parts of text, usually for titles.
+
+### Types
+
+#### h1 -
+`<h1>(text)</h1>`
+
+#### h2 -
+`<h2>(text)</h2>`
+
+#### h3 -
+`<h3>(text)</h3>`
+
+#### h4 -
+`<h4>(text)</h4>`
+
+#### h5 -
+`<h5>(text)</h5>`
+
+#### h6 -
+`<h6>(text)</h6>`
+
+## Lists 
+
+### Unordered Lists 
+
+#### Reason - 
+Shows text in a Bullited List
+
+#### Usage - 
+`<ul>(tags)</ul>`
+
+### Ordered Lists
+
+#### Reason - 
+Shows text in a Numbered List
+
+#### Usage - 
+`<ol>(tags)</ol>`
+
+### List Items
+
+#### Reason - 
+The Text in each List
+
+#### Usage - 
+`<li>(text)</li>`
+
+## Links
+
+#### Reason - 
+Connecting Pages in HTML or redirects to other external pages.
+
+#### Usage - 
+`<a href="url">(text)</a>`
+
+## Form
+
+#### Reason - 
+Create a submit form for buttons and text areas.
+
+#### Usage - 
+`<form action="url">(tags)</form>`
+
+## Buttons
+
+#### Reason - 
+A Clickable Area
+
+#### Usage - 
+`<button>(text)</button>`
+
+<!-- # CSS -->
+
```

Comment:
same as above

Reply:
All the blocking just seems a bit well, blocky, I think the little pop without any full line blocks, makes it look better, if you still think blocks would look better I can change it.